src/string_radix_sort.jl

```diff
@@ -0,0 +1,233 @@
+import Base: Forward, ForwardOrdering, Reverse, ReverseOrdering, Lexicographic, LexicographicOrdering, sortperm, Ordering, 
+            setindex!, getindex, similar
+import SortingAlgorithms: RadixSort, RadixSortAlg
+"""
+    load_bits([type,] s, skipbytes)
+
+Load the underlying bits of a string `s` into a `type` of the user's choosing.
+The default is `UInt`, so on a 64 bit machine it loads 64 bits (8 bytes) at a time.
+If the `String` is shorter than 8 bytes then it's padded with 0. 
+
+Assumes machine is little-endian
```

Comment:
Is that true even with the `ntoh` calls?

Reply:
It's not true. Have removed